### PR TITLE
Fix: #3411 Merge administrator_controls_download_permissions_view into single xml file.

### LINKED ISSUE #3411: Merge administrator_controls_download_permissions_view into single xml file.
Currently there are three versions of `administrator_controls_download_permissions_view.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by Akshatkamboj14**
## Explanation
FIXES #3411

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

| BEFORE  | AFTER  |
| --------- | -------- |
| Screenshot_20220213_000010 | Screenshot_20220215_124149 |
| Screenshot_20220215_122455 | Screenshot_20220215_124233 |
|Screenshot_20220213_001428|Screenshot_20220213_111012|
|Screenshot_20220213_114014|Screenshot_20220209_213851|

### CODE REVIEW

**Comment by Akshatkamboj14** [comment]
@rishidyno please review it

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 1]
```diff
@@ -420,4 +420,14 @@
...
+  <!-- AdministratorControlDownloadPermissionsView -->
+  <dimen name="administrator_controls_download_permissions_view_download_permissions_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_download_permissions_text_view_padding_end">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_topic_update_on_wifi_constraint_layout_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_topic_update_on_wifi_constraint_layout_padding_end">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_topic_update_on_wifi_widget_switch_compat_margin_end">4dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">4dp</dimen>
+
```
Comment (rishidyno) [inline app/src/main/res/values-sw600dp-land/dimens.xml]: Please remove this extra line.

File: app/src/main/res/values-sw600dp-port/dimens.xml [hunk 2]
```diff
@@ -419,4 +419,14 @@
...
+  <!-- AdministratorControlDownloadPermissionsView -->
+  <dimen name="administrator_controls_download_permissions_view_download_permissions_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_download_permissions_text_view_padding_end">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_topic_update_on_wifi_constraint_layout_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_topic_update_on_wifi_constraint_layout_padding_end">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_topic_update_on_wifi_widget_switch_compat_margin_end">4dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">16dp</dimen>
+  <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">4dp</dimen>
+
```
Comment (rishidyno) [inline app/src/main/res/values-sw600dp-port/dimens.xml]: Ditto.

**Review by rishidyno — CHANGES_REQUESTED** [review]

**Comment by rishidyno** [comment]
And try to fix the failing check. I think this should not happen but seems to happen in other ps as well. Just commit the requested changes and will see what happens.

**Comment by rishidyno** [comment]
Also update your project.

**Comment by Akshatkamboj14** [comment]
done @rishidyno

**Review by rishidyno — CHANGES_REQUESTED** [review]
@Akshatkamboj14 I think this pr is good to go, but always use same device to take SS. I suggest updating SS for mobile-land. So that we can be sure that there are no changes in UI before and after. Also, use an emulator without round corners. Thanks.

**Comment by Akshatkamboj14** [comment]
hey, @rishidyno I have updated the ss of mobile ones with the new emulator.

**Comment by rishidyno** [comment]
@Akshatkamboj14 also resolve the merge conflict

**Comment by rishidyno** [comment]
Also please keep the `Assignees` section up-to-date, so that we can know whether to review this PR or not.

**Comment by rishidyno** [comment]
FYI the person who has to review should be assigned

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 3]
```diff
@@ -419,8 +419,18 @@
   <dimen name="revision_card_fragment_padding_bottom">148dp</dimen>
   <dimen name="revision_card_fragment_layout_text_margin_top">32dp</dimen>
   <dimen name="revision_card_fragment_layout_button_margin_top">24dp</dimen>
-
+
```
Comment (rishidyno) [inline app/src/main/res/values-sw600dp-land/dimens.xml]: PTAL -->  this can be corrected.

**Review by rishidyno — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-sw600dp-land/dimens.xml (hunk 3 above)
Comment (Akshatkamboj14) [reply app/src/main/res/values-sw600dp-land/dimens.xml]: done

**Comment by rishidyno** [comment]
LGTM thanks @Akshatkamboj14 .

**Review by rt4914 — APPROVED** [review]
This is really clean code. Great work @Akshatkamboj14 and @rishidyno 

LGTM

**Pull request merged**